bin/katello-enabled-repos-upload

```diff
@@ -18,7 +19,17 @@ def main():
     (options, args) = parse_args()
     if options.force:
         EnabledRepoCache.remove_cache()
-    report = EnabledReport(REPOSITORY_PATH)
+
+    if path.exists('/usr/bin/yum'):
+        tool = TOOL_YUM
+        repo_path = REPOSITORY_PATH
+    elif path.exists('/usr/bin/zypper'):
+        tool = TOOL_ZYPPER
+        repo_path = ZYPPER_REPOSITORY_PATH
+    else:
+        raise IOError('Neither yum nor zypper repo file found')
+
+    report = EnabledReport(repo_path, tool)
     upload_enabled_repos_report(report, options.force)
```

Comment:
Take a look at this section of the code: https://github.com/Katello/katello-host-tools/blob/31469a1a866a0979afadec364c0d2000dc55bb49/src/katello/repos.py#L46-L47

`combined_profiles_enabled` means "does subscription-manager support uploading both package and repository profile?"  In this case we do not send our own report, but let subman generate it. But notice: in this case we've already created a report that we will not use. We can make an improvement here, simple solution being:

In `bin/katello-enabled-repos-upload` we can check `combined_profiles_enabled` and generate the report only if we will actually send it. Then repos.py can be updated:
```
- elif combined_profiles_enabled():
+ elif report is None:
```

Zypper plugin can be updated similarly.